onnxruntime/core/language_interop_ops/torch/torch_proxy.cc

```diff
@@ -171,6 +171,12 @@ void InvokeRunner(
   // first element.
   for (; i < static_cast<size_t>(PyTuple_Size(result_ptr.get())); ++i) {
     PyObject* dl_tensor_pointer = PyTuple_GetItem(result_ptr.get(), i);
+    if (dl_tensor_pointer == Py_None) {
```

Comment:
We previously filter out all `None`s from Python side. Does this change means we don't filter out `None`s anymore?

Reply:
Yes, we don't filter them out, because None can be in any places of the output lists. We need map them correctly into PythonOp kernels's outputs.